test/integration_test/backup_test.go

```diff
@@ -0,0 +1,137 @@
+// +build integrationtest
+
+package integrationtest
+
+import (
+	"fmt"
+	"testing"
+
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	"github.com/portworx/sched-ops/k8s"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	instanceID = "basic"
+	appKey     = "mysql-app-to-backup"
+)
+
+func testBackup(t *testing.T) {
+	t.Run("backupTest", backupRestoreDeleteTest)
+}
+
+var backupLocationMap = map[string]storkv1.BackupLocationType{
+	"s3":     storkv1.BackupLocationS3,
+	"azure":  storkv1.BackupLocationAzure,
+	"google": storkv1.BackupLocationGoogle,
+}
+
+func triggerBackupRestoreTest(
+	t *testing.T,
+	instanceID string,
+	appKey string,
+	appBackupKey []string,
+	appRestoreKey []string,
+) {
+	var err error
+	// Reset config in case of error
+	defer func() {
+		err = setRemoteConfig("")
+		require.NoError(t, err, "Error resetting remote config")
+	}()
+
+	ctxs, _ := createAppForBackup(t, instanceID, appKey)
```

Comment:
There is a `createApp()` in `snapshot_test.go`, you should just use that. I'm moving it to `common_test.go` in #418

Reply:
Fixed.